openmm-rbfe/ApplyingProtocolToNetworkQuickrunDemo.ipynb

```diff
@@ -0,0 +1,520 @@
+    "    )\n",
+    "    stateB_complex = openfe.setup.ChemicalSystem(\n",
+    "        components={'solvent': solvent,\n",
+    "                    'ligand': ligB,\n",
+    "                    'protein': protein},\n",
+    "    \n",
+    "    )\n",
+    "    # similarly define the solvent leg\n",
+    "    stateA_solvent = openfe.setup.ChemicalSystem(\n",
+    "        components={'solvent': solvent, 'ligand': ligA}\n",
+    "    )\n",
+    "    stateB_solvent = openfe.setup.ChemicalSystem(\n",
+    "        components={'solvent': solvent, 'ligand': ligB}\n",
+    "    )\n",
+    "    \n",
+    "    complex_trans = openfe.setup.Transformation(\n",
+    "        stateA=stateA_complex,\n",
+    "        stateB=stateB_complex,\n",
+    "        mapping={'ligand': edge},\n",
+    "        protocol=prot,\n",
+    "    )\n",
+    "    solvent_trans = openfe.setup.Transformation(\n",
+    "        stateA=stateA_solvent,\n",
+    "        stateB=stateB_solvent,\n",
+    "        mapping={'ligand': edge},\n",
+    "        protocol=prot,\n",
+    "    )\n",
+    "    \n",
+    "    transformations.append(complex_trans)\n",
+    "    transformations.append(solvent_trans)\n",
+    "\n",
+    "print(f\"From {len(network.edges)} edges we have defined {len(transformations)} Transformations\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "09acea93",
+   "metadata": {},
+   "source": [
+    "## 3) Save, load and execute your Transformation elsewhere\n",
+    "\n",
+    "With the chemistry and method defined for many edges, we are now ready to perform the calculations.\n",
+    "Inevitably, we will want to farm out these jobs to different resources.\n",
+    "To do this we will need to store the data on disk,\n",
+    "transmit it elsewhere, then run the stored Transformation.\n",
+    "\n",
+    "A simple way to save a `Transformation` to disk is to use the `.dump()` method.\n",
+    "\n",
+    "\n",
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "    <b>Note</b>\n",
+    "    \n",
+    "* This section is provided as a basic way of trying out the OpenFE tools.  A more comprehensive method for executing many jobs is in the works.\n",
+    "* Storing entire campaigns of work this way is inefficient, a more comprehensive and robust method for archiving data is being developed.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "f328af99",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# pick an arbitrary Transformation to demonstrate with\n",
+    "trans = transformations[0]\n",
+    "\n",
+    "trans.dump('out.json')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b4faf42c",
+   "metadata": {},
+   "source": [
+    "We can see that this has created a file in our local directory.\n",
+    "For users of Gromacs, this json is similar to a compiled `.tpr` file, storing a prepared simulation which is ready to be executed.\n",
```

Comment:
Because the force field info is inside the Settings blob, while the parametrisation hasn't happened, it should be deterministically defined at this point, so it is more similar to tpr than an openmm System.

Reply:
A tpr already includes the full parametrisation, whilst this only includes a link to the parameters. This is closer to an ITP than a TPR

Reply:
(i.e. if you were to take this blob to a machine which didn't have openmmforcefields installed or a different version, you could technically expect to get a different set of parameters applied)

Reply:
Semantics we can argue over later.